README.md

```diff
@@ -110,37 +98,16 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
-## Environment Variables:
-
-Some plugins require environment variables to connect to their external systems, such project IDs or API keys.
-Follow instructions from individual plugin documentations, such as:
-
-* [ape-alchemy](https://github.com/ApeWorX/ape-alchemy/blob/main/README.md#quick-usage)
-* [ape-infura](https://github.com/ApeWorX/ape-infura#readme)
+## Ape Modular Plugin Sytem:
 
-Generally, set environment variables by doing the following:
+One of the best feature of Ape is our modular plugin system. Our plugins is the way we want you to have most interoperable experience with Web Ecosystem.
 
-```bash
-# Used by the `ape-infura` plugin
-export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>
-# Used by the `ape-alchemy` plugin
-export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
-```
-
-Place these in environment files, such as your `.bashrc` or `.zshrc`.
-
-## Quick Usage
-
-Use `-h` to list all the commands:
-
-```bash
-ape -h
-```
+[Plugin list]()
 
-### Projects
+### Quick Start 
 
-When using Ape, you generally will work with a project.
-To quickly get started using ape, generate a project using the `ape init` command:
+After you installed Ape and you want to get started, you generally will work with a project and need the proper scaffolding. 
```

Comment:
This sentence kind of seems like a more verbose way of saying what is already there. Perhaps could use some revising.